charts/incubator/deemix/values.yaml

```
@@ -0,0 +1,36 @@
+image:
+  repository: tccr.io/truecharts/deemix
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:61eda30552bca8c1fa1dd04f70adb20bf9d6ed56
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+
+service:
+  main:
+    enabled: true
+    ports:
+      main:
+        port: 6595
+        targetPort: 6595
+
+probes:
+  liveness:
+    enabled: true
+  readiness:
+    enabled: false
+  startup:
+    enabled: false
```

```suggestion
probes:
  readiness:
    enabled: false
  startup:
    enabled: false
```
You can remove this as it's enabled by default.

Is there a reason for disabling the other 2?

---

Not really, I was unsure how it would behave.

---

Best to remove all of them so they are enabled by default, and disable/adjust if need. (based on test results)